### ISSUE TITLE
Unsupported platform macOS with python in native build

### ISSUE THREAD

**Issue by LilPoppy** [oracle/graalpython#258]
Hi this issue has been pointed out from [issue 4192](https://github.com/oracle/graal/issues/4192)

[CI](https://github.com/SpamanDev/StoreChain-Server/runs/5176598743?check_suite_focus=true#step:9:3610)

**Comment by msimacek** [comment]
Hi @LilPoppy, we support and test Mac OS in GraalPython and never encountered such error. The error doesn't seem to be related to python and we don't use the class shown in the stacktrace, I think the problem will be a different component that you're using. Are you able to build your application without python on Mac?

**Comment by LilPoppy** [comment]
> Hi @LilPoppy, we support and test Mac OS in GraalPython and never encountered such error. The error doesn't seem to be related to python and we don't use the class shown in the stacktrace, I think the problem will be a different component that you're using. Are you able to build your application without python on Mac?

Yes, without python will be successfully build [CI](https://github.com/SpamanDev/StoreChain-Server/runs/5279127594?check_suite_focus=true) and [pom.xml](https://github.com/SpamanDev/StoreChain-Server/blob/8c620ee325677662d529d3773bcce99e08da39a2/pom.xml#L376)
